src/main/java/org/jitsi/videobridge/SctpConnection.java

```diff
@@ -1118,9 +1121,6 @@ public void run()
                     }
                 }
 
-                if (sctpSocket == null)
-                    break;
-
```

Comment:
I will close this PR. I agree that we should refactor this anyway, and doing so is on the roadmap, so I'll leave it for now.